src/porepy/numerics/parabolic.py

```diff
@@ -64,30 +65,41 @@ def space_disc(self):
     problem.solve()
     '''
 
-    def __init__(self, gb, physics='transport', **kwargs):
+    def __init__(self, gb, physics='transport',time_step=1.0, end_time=1.0, **kwargs):
         self._gb = gb
+        self.is_GridBucket = isinstance(self._gb, GridBucket)
         self.physics = physics
-        self._data = dict()
+        self._data = kwargs.get('data', dict())
+        self._time_step = time_step
+        self._end_time = end_time
         self._set_data()
+
         self._solver = self.solver()
-        self._solver.parameters['store_results'] = True
+        self._solver.parameters['store_results'] = False
 
```

Comment:
Can we remove this parameter variable?

Reply:
We don't use it, do we? If not, then clearly.